src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -31,6 +32,7 @@
 
 public record CreateCollectionOperation(
     CommandContext commandContext,
+    DeleteCollectionOperation deleteCollectionOperation,
```

Comment:
Instead of passing via constructor, this could/should be constructed as needed, as `commandContext` and collection name are known within code.

Reply:
Make Sense, thanks!